README.md

```diff
@@ -135,34 +136,11 @@ img.get_channel_names()  # returns a list of string channel names found in the m
 [napari](https://github.com/Napari/napari) is a fast, interactive, multi-dimensional
 image viewer for python and it is pretty useful for imaging data that this package
 tends to interact with.
-```python
-from aicsimageio import AICSImage
-
-# Get an AICSImage object
-img = AICSImage("my_file.tiff")
-img.view_napari()  # launches napari GUI and viewer
-```
 
 We have also released
 [napari-aicsimageio](https://github.com/AllenCellModeling/napari-aicsimageio), a plugin
-that allows use of all the functionality described here, but in the `napari` default
-viewer itself.
-
-
-## Performance Considerations
```

Comment:
this is just gone?  shouldn't there be a few hints here to guide the usage of get_image_data vs get_image_dask_data?

Reply:
I figured that the "Quickstart Notes" section was good enough.

Reply:
I agree with that info but I like the FAQ-like presentation here.  I can easily see which cases apply to me.

Reply:
Will add it back in reduced form.